src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -225,4 +227,10 @@ export class PoolListComponent implements OnInit {
   getPoolDetails(pool: object) {
     return _.omit(pool, ['cdExecuting', 'cdIsBinary']);
   }
+
+  getSelectionTiers() {
+    let cacheTierIds: number[];
+    cacheTierIds = this.selection.hasSingleSelection ? this.selection.first()['tiers'] : [];
```

Comment:
As you are not reassigning `cacheTierIds` you should use `const` instead.
 
```suggestion
    const cacheTierIds = this.selection.hasSingleSelection ? this.selection.first()['tiers'] : [];
```